backend/infra/infra/cw_core_stack.py

```diff
@@ -417,7 +417,8 @@ def create_cw_api_gw(self):
             scope=self,
             id=f"{self.stack_env}-cw-auth-api",
             description="CarWorld API Gateway",
-            cors_preflight=apigw.CorsPreflightOptions(
+            #TODO: look into this
```

Comment:
Nit: Weird line spacing introduced?

Request: Can we improve the this comment with more context if you'd like to leave it in?

Reply:
Just edited the first commit in this PR to fix these two issues!